src/api/app/controllers/webui/obs_factory/application_controller.rb

```diff
@@ -2,7 +2,7 @@ module Webui::ObsFactory
   class ApplicationController < ::Webui::WebuiController
     layout 'webui/obs_factory/application'
 
-    rescue_from ::ObsFactory::OpenqaApi::OpenqaFailure do |ex|
+    rescue_from ::ObsFactory::OpenqaApi::OpenqaFailure do |_ex|
```

Comment:
what about removing the variable instead? :thinking:

Reply:
I'm fine with that too. But that was what the auto-correct did. I will remove the variable.

Reply:
Variable removed. @Ana06 please re-review.